frontend/apps/erd-web/package.json

```diff
@@ -24,6 +24,7 @@
     "fmt:biome": "biome check --write --unsafe .",
     "lint": "pnpm run '/^lint:.*/'",
     "lint:biome": "biome check .",
+    "postinstall": "cp ../../packages/db-structure/src/parser/schemarb/prism.wasm prism.wasm",
```

Comment:
Here it is ran with postinstall.
There is a possible way to include it in the webpack bundle with [CopyWebpackPlugin](https://webpack.js.org/plugins/copy-webpack-plugin/), but the new [turbopack](https://turbo.build/pack/docs) for Next.js that we are currently using is not compatible with the webpack plugin, so I did not adopt it.